api/client/lib.py

```diff
@@ -538,26 +538,26 @@ def lookup_belongs(access_token, api_host, entity_type, entity_id):
 
 
 def get_geo_centre(access_token, api_host, region_id):
-    url = '/'.join(['https:', '', api_host, 'v2/geocentres?regionIds=' +
-                    str(region_id)])
+    url = '/'.join(['https:', '', api_host, 'v2/geocentres'])
     headers = {'authorization': 'Bearer ' + access_token}
-    resp = get_data(url, headers)
+    resp = get_data(url, headers, {'regionIds': region_id})
     return resp.json()['data']
 
 
 @memoize(maxsize=None)
-def get_geojsons(access_token, api_host, region_id, descendant_level=None):
-    url = '/'.join(['https:', '', api_host, 'v2/geocentres?includeGeojson=True&regionIds={}'.format(
-        region_id)])
+def get_geojsons(access_token, api_host, region_id, descendant_level, zoom_level):
```

Comment:
These lib functions, `get_geo_centre`, `get_geojsons`, and `get_geojson` are not covered by any tests, and since this PR is adding code to them, it has decreased test coverage a bit. Add these to `lib_test.py`?

Reply:
yeah you're right. working on it now